datastream-server/src/main/java/com/linkedin/datastream/server/zk/ZkAdapter.java

```diff
@@ -1765,6 +1770,10 @@ void onSessionExpired() {
     // cancel the lock clean up
     _orphanLockCleanupFuture.cancel(true);
     onBecomeFollower();
+    if (_listener != null) {
+      _listener.onSessionExpired();
+    }
+    // currently it will try to disconnect and fail. TODO: fix the connect and listen to handleNewSession.
```

Comment:
nit: do you still need this comment? Won't the exiting comment be sufficient?

Reply:
I have a TODO marker which makes it explicit.